hack/godep-save.sh

```diff
@@ -33,35 +35,35 @@ REQUIRED_BINS=(
   "./..."
 )
 
+STAGING_REPOS=(
+  "k8s.io/api"
+  "k8s.io/client-go"
+  "k8s.io/apiserver"
+  "k8s.io/apimachinery"
+  "k8s.io/kube-aggregator"
+  "k8s.io/apiextensions-apiserver"
+  "k8s.io/sample-apiserver"
+  "k8s.io/metrics"
+)
+
 pushd "${KUBE_ROOT}" > /dev/null
+  # sanity check that staging directories do not exist in GOPATH
```

Comment:
We had reports a couple of times from people who failed with non-trivial GOPATHs. I would better check that.